mlir/lib/Dialect/Rock/utility/transformMapUtils.cpp

```diff
@@ -322,16 +322,27 @@ void findCountiguousGroupsUnmerge(const ArrayRef<uint32_t> upperDims,
         std::is_sorted(dimPosition.begin(), dimPosition.end())) {
 
       uint32_t lowerDim = groupCandidate.back();
+      size_t lowerDimPosInMerge = dimPosition.back();
```

Comment:
Nit: why not `fastestDimPosInMerge` (since you use ` slowestDimPosInMerge = dimPosition.front()`)?

Reply:
Existing code naming, but I might as well clean that up while I'm here